app/src/main/java/ar/scacchipa/twittercloneapp/data/repository/ILoggedUserRepository.kt

```diff
@@ -0,0 +1,9 @@
+package ar.scacchipa.twittercloneapp.data.repository
+
+import ar.scacchipa.twittercloneapp.data.model.UserData
+
+interface ILoggedUserRepository {
+    suspend fun refreshLoggedUser(): Boolean
```

Comment:
The AuthorizationUseCase class was updated.
AuthorizationUseCase check if it could not refreshLoggedUser and returns ResponseDomain.Error() and the app return to Login Fragment.